lib/3scale/backend/storage.rb

```diff
@@ -198,9 +201,8 @@ def cfg_sentinels_handler(options)
                   raise InvalidURI.new("(sentinel #{sentinel.inspect})",
                                        'no host given')
                 end
-                sentinel.fetch(:port) do
-                  raise InvalidURI.new("(sentinel #{sentinel.inspect})",
-                                       'no port given')
+                if !sentinel[:port] || sentinel[:port] == ""
```

Comment:
`sentinel[:port] ||= DEFAULT_SENTINEL_PORT`

Reply:
In that case if sentinel[:port] is "", sentinel[:port] would have the value "":

```
irb(main):003:0> sentinel = { port: "" }
=> {:port=>""}
irb(main):004:0> sentinel[:port] ||= 26379
=> ""
irb(main):005:0> sentinel[:port]
=> ""
```

The idea here was to assign the default sentinel port if originally port is the empty string

Reply:
I would not expect to be empty string, nor empty array or empy hash or any other object. Up to you.

Reply:
So, in this case we will let the client crash if the user configures {}, "", or [] so they can detect the mistake :+1: